pkg/apis/workload/v1alpha1/types.go

```diff
@@ -119,12 +144,12 @@ type ServiceBusinessIndicatorSpec struct {
 	Indicators []Indicator `json:"indicators,omitempty"`
 }
 
-type TargetIndicatorName string
```

Comment:
changing type name, will it cause modifications in codes using this type?

Reply:
yes, i will change the code in other mr